README.md

```diff
@@ -242,6 +242,31 @@ const client = new faunadb.Client({
 })
 ```
 
+## Knowing issues
```

Comment:
## Known issues

### Using with Cloudflare Workers

Cloudflare Workers has neither XMLHttpRequest nor fetch in global scope.
Therefore, cross-fetch package is unable to inject fetch, and throws an error.
fetch function is injected via closure, so the workaround would be to pass
the fetch objects when initiating the FaunaDB client config. Cloudflare also 
doesn't support AbortController, which terminates requests as well as streams. 
Here is a workaround:

Reply:
just some minor grammatical edits

Reply:
done